Source/SpatialGDK/Private/SpatialInteropPipelineBlock.cpp

```diff
@@ -187,6 +187,21 @@ void USpatialInteropPipelineBlock::LeaveCriticalSection()
 		RemoveEntityImpl(PendingRemoveEntity);
 	}
 
+	// Notify authority changes
+	for (auto& PendingAuthorityChange : PendingAuthorityChanges)
+	{
+		// Use position component as a proxy for overall actor authority
+		if (PendingAuthorityChange.Key.ComponentId == UPositionComponent::ComponentId)
```

Comment:
You can use the `View` to directly get authority rather than using another component.

Reply:
Chatted offline -- current implementation is okay.